### ISSUE TITLE
Document requirements for Reactome models as Templates use case

### ISSUE THREAD

**Issue by goodb** [geneontology/pathways2GO#78]
Complete this issue with a user story (with illustrations) for how we would like to see curators make use of **Reactome models as templates for new models**.  

Answer:
In this use context, what is unique about the reactome models as opposed to other models in the go-cam repository?  (Is this a specific issue for the pathways2go project or a more general Noctua issue?) 

What goals would a curator be seeking to achieve with this workflow?

What sequence of steps would we expect the curator to perform?

What is the end product of the workflow?

**Comment by goodb** [comment]
@ukemi @deustp01 my understanding from the meeting this week is that this is going to be deferred - most likely rolled into a grant proposal.  Shall we close this or move into a new project?

**Comment by deustp01** [comment]
Yes, defer for future is my recollection, and it was clear from the NYU discussion that figuring out how to proceed would probably require discussions with a range of users and development of a range of use cases, so this moves into a new large project. Maybe a sentence of discussion, and at least part of it is a feature of the grant.

**Comment by ukemi** [comment]
Yes. I think this would be rolled into a larger discussion of model copy/templates and the Reactome usage will require further resources. 

Seems like this is also entwined with the larger history-tracking discussions that are about to take the limelight.

Pinging @vanaukenk  to see if she has comments.

**Comment by ukemi** [comment]
See this document:
https://docs.google.com/document/d/1Vrc-oYZDaJFqYLotA2ce_cYXHWqDwwEvK9HrT-3JQcI/edit
